blog/docs/steering/knative-and-endusers.md

```diff
@@ -0,0 +1,133 @@
+---
+Title: "End Users and Knative"
+Author: "[Murugappan Chetty](https://github.com/itsmurugappan)"
+Author handle: https://twitter.com/itsmurugappan
+date: 2022-03-08
+description: Knative from the lens of an end user
+folder with media files: "images/enduser"
+blog URL: "https://itsmurugappan.medium.com"
+labels: Article
+type: "blog"
+---
+
+**_Author: [Murugappan Chetty](https://twitter.com/itsmurugappan) (Optum,
+representing end users @ Knative Steering Committee)_**
+
+Knative is an open-source community that values end-user
+contributions/interactions highly, this is evident in PRs, monthly meetups,
+slack interactions, etc. To top it all, recently Knative opened an end-user seat
+in the steering committee. This is very significant for a couple of reasons
+
+- Instills confidence in end-user organizations to use Knative in production
+- Very few OSS communities have end-user representation in the steering
+  committee
+
+I am pleased to share that I have been elected as the first end-user
+representative in the steering committee. In this blog, I would like to share my
+organization’s Knative experience so far.
+
+## Why Knative?
+
+First things first, Knative was not picked for being the next shiny object. The
+whole process was organic. The following factors led to the need for a
+Kubernetes abstraction.
+
+- Mature container ecosystem.
+- Serverless workloads
+- Need for better developer experience than vanilla Kubernetes (deployment +
+  service + ingress + ...)
+
+The situation was ripe for building a new platform on top of Kubernetes. Before
+doing that we wanted to check if we could leverage an existing open-source
+solution, there were a handful of them at various maturity levels, Knative was
+the one that stood out for us
+
+- Best scaling performance compared to all its peers.
+- Developer experience (single knative service resource yields Kubernetes
+  deployment + service + ingress + certificate + custom domain).
+- Robust Primitives for building a developer platform.
+- OSS Community with a good governance model
+- Not backed by a single vendor
```

Comment:
Nit, I wonder if this would read better as a non-negative? Something like:
Backed by multiple vendors